src/LanguageServer/Protocol/Handler/Extensions/ExtensionRegisterParams.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Json.Serialization;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Handler.Extensions;
+
+/// <summary>
+/// Parameters for the roslyn/extensionRegister request.
+/// </summary>
+/// <param name="AssemblyFilePath">Full path to the assembly that contains the message handlers to register.</param>
+internal readonly record struct ExtensionRegisterParams(
```

Comment:
seems uneeded.  we can just pass the `string assemblyFilePath`.

Reply:
Same as before - params objects make this much easier to handle new fields if we need them

Reply:
ok.